tests/target/issue_3033.rs

```diff
@@ -1,2 +1,12 @@
-use dom::bindings::codegen::Bindings::BluetoothRemoteGATTServerBinding::BluetoothRemoteGATTServerBinding::
-    BluetoothRemoteGATTServerMethods;
+use dom::bindings::codegen::Bindings::BluetoothRemoteGATTServerBinding::
+    BluetoothRemoteGATTServerBinding::BluetoothRemoteGATTServerMethods;
```

Comment:
Exceeds 100 chars. It was preserved because the format failed.

Reply:
Given that this changes existing behavior these changes need to be [version gated](https://github.com/rust-lang/rustfmt/blob/master/Contributing.md#version-gate-formatting-changes)

Reply:
Ah, sorry. I missed the comment. Fixed.